packages/arb-token-bridge-ui/src/hooks/useTransactionHistory.ts

```diff
@@ -733,16 +733,21 @@ export const useTransactionHistory = (
         return
       }
 
-      // ETH deposit
-      if (tx.assetType === AssetType.ETH) {
+      const isDifferentDestinationAddress =
+        tx.sender &&
+        tx.destination &&
+        tx.sender.toLowerCase() !== tx.destination.toLowerCase()
```

Comment:
i prefer this since `tx.sender` and `tx.destination` are not used below so we didn't need specific `undefined` checks

<img width="544" alt="image" src="https://github.com/user-attachments/assets/1bc1c21f-6a53-4517-bd2b-a28719f7e4bf">

see
<img width="633" alt="image" src="https://github.com/user-attachments/assets/7606e1c6-edf0-4e68-b551-e593b838a9b2">

Reply:
But then if e.g. only destination address is undefined we will end up with:
```typescript
'0xabcd' !== undefined
```
Which returns true, and `isDifferentDestinationAddress` will be true in this case even though it shouldn't be.

To fix the types I can check for `typeof tx.sender !== 'undefined'` and `typeof tx.destination !== 'undefined'`